client/src/components/ReviewsRatings/NewReview.jsx

```diff
@@ -59,11 +58,10 @@ const NewReview = ({ reviews, productId, reviewChar }) => {
     setReviewCharValues(obj);
   };
 
+
+
   return (
     <div>
-      <Button variant="outline-dark">More Reviews</Button>
-      <Button variant="outline-dark" onClick={handleShow}>Add a Review +</Button>
-
       <Modal show={show} onHide={handleClose} centered>
```

Comment:
Add some white space on either side of the params to show/onHide for visual consistency.